vms/platformvm/service.go

```diff
@@ -2131,8 +2131,10 @@ type GetConfigurationReply struct {
 	MinConsumptionRate json.Uint64 `json:"minConsumptionRate"`
 	// The maximum consumption rate
 	MaxConsumptionRate json.Uint64 `json:"maxConsumptionRate"`
-	// The supply cap for the native tolen (AVAX)
+	// The supply cap for the native token (AVAX)
```

Comment:
CAM not AVAX

Reply:
We handle both CAM and AVAX in camino, depending network.